manifests/n/NimbusWeb/NimbusNote/10.0.11/NimbusWeb.NimbusNote.locale.en-US.yaml

```diff
@@ -0,0 +1,25 @@
+# Automatically updated by the winget bot at 2022/Apr/23
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: NimbusWeb.NimbusNote
+PackageVersion: 10.0.11
+PackageLocale: en-US
+Publisher: NimbusWeb
+PublisherUrl: https://nimbusweb.me/index.php
+PublisherSupportUrl: https://nimbusweb.me/customer-support.php
+PrivacyUrl: https://nimbusweb.me/privacy.php
+Author: NimbusWeb
+PackageName: Nimbus Note
+PackageUrl: https://nimbusweb.me
+License: Copyright (C) 2020 NimbusWeb
+LicenseUrl: https://nimbusweb.me/terms-and-conditions.php
+Copyright: Copyright (C) 2020 NimbusWeb

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate